Task2/Task2BracketsBalance.c

```diff
@@ -0,0 +1,93 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include "../stack/stack.h"
+
+typedef enum Error
+{
+    MemoryAllocationError,
+    Balanced,
+    Unbalanced
+} Error;
+
+bool isMatchingBrackets(char openingBracket, char closingBracket)
+{
+    if (openingBracket == '(' && closingBracket == ')')
+        return true;
+    if (openingBracket == '{' && closingBracket == '}')
+        return true;
+    if (openingBracket == '[' && closingBracket == ']')
+        return true;
+    return false;
```

Comment:
```suggestion
    return openingBracket == '(' && closingBracket == ')' ||
           openingBracket == '{' && closingBracket == '}' ||
           openingBracket == '[' && closingBracket == ']';
```
Само условие условных операторов уже является искомым результатом